odgovlt.py

```diff
@@ -214,6 +217,35 @@ def sync_organization(self, istaiga_id, conn):
         organization_data['id'] = ckan_organization['id']
         return organization_data
 
+    def sync_group(self, grupe_id, conn):
+        self.api = CkanAPI()
+        self.importbot = self.sync_importbot_user()
+
+        group = conn.execute(sa.select([self.t.kategorija]).where(self.t.kategorija.c.ID == grupe_id)).fetchone()
```

Comment:
I think, it is better to define `self.t` in `_connect_to_database` method, now it looks, that it is defined here by accident.